starboard/build/config/modular/BUILD.gn

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+config("default_cpp_standard") {
```

Comment:
Is this something we'll eventually want to have on other platforms, or will we only ever need/want it for modular builds?

Reply:
this wont be the case for all platforms : think clang3-9 ,  host toolchain which build with cpp14.

Reply:
I mean will we eventually want to extract the standard from the platform's configuration and have it in a platform-agnostic spot like this? Or will we only ever want to extract it from modular builds?

Reply:
i think it'll be good to extract it from all non-third party code and keep it in a single file with multiple configs: cpp14, cpp17, cpp20 . it does cause confusion to have it part of the platform flags - the flags end up overlapping - for ex; for llvm : cpp17 came from platform config whereas cpp20 came from llvm repo.

Reply:
Okay, then let's add it to a more general location, i.e. `//starboard/build/config/BUILD.gn`

Reply:
good idea, done